lib/plug/adapters/test/conn.ex

```diff
@@ -163,6 +171,15 @@ defmodule Plug.Adapters.Test.Conn do
 
   defp stringify_kv({k, v}), do: {to_string(k), stringify_params(v)}
 
+  defp stringify_all_params([{_, _} | _] = params), do: Enum.into(params, %{}, &stringify_pair/1)
```

Comment:
Why do we need both stringify_params and stringify_all_params?

Reply:
- `stringify_params` recursively string-ifies the only keys
- `stringify_all_params` recursively string-ifies keys _and_ values, and is only used for GET/HEAD requests when params is enumerable.

We need both functions because we don't want to stringify values for body params, but we do want to stringify values for query params.

Maybe renaming the functions `stringify_body_params` and `stringify_query_params` might be clearer?  I hesitated to refer to these as _query_ params, since we don't actually touch the query params portion of the Conn struct.

Reply:
I updated `stringify_params/1` to `stringify_params/2` using a value function, as discussed. wdyt?